hadoop-common-project/hadoop-common/src/main/java/org/apache/hadoop/ha/NodeFencer.java

```diff
@@ -46,7 +46,7 @@
  * The fencing methods that ship with Hadoop may also be referred to
  * by shortened names:<br>
  * <ul>
- * <li><code>shell(/path/to/some/script.sh args...)</code></li>
+ * <li><code>shell(/path/to/some/script.sh args...)</code> (see {@link ShellCommandFencer}) </li>
```

Comment:
can we remove "</li>" at the end of the line, to be consistent with the other two?

Reply:
@xinglin , thanks for your comments and suggestions.
I will improve it soon.